src/web/app/pages-instructor/instructor-student-records-page/__snapshots__/instructor-student-records-page.component.spec.ts.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InstructorStudentRecordsPageComponent should snap with default fields 1`] = `
+<tm-instructor-student-records-page
+  courseId="undefined"
+  photoUrl={[Function String]}
+  route={[Function ActivatedRoute]}
+  sessions={[Function Array]}
+  statusMessageService={[Function StatusMessageService]}
+  studentEmail="undefined"
+  studentProfile={[Function Object]}
+  studentProfileService={[Function StudentProfileService]}
+>
+  <h1>
+    's Records
+    <small
+      class="text-muted"
+    >
+       - 
+    </small>
+  </h1><tm-student-profile
+    ng-reflect-photo-url="http://localhost:8080/webapi/s"
```

Comment:
Looks like snapshot testing does not capture full value.

Reply:
I tried doing this in the past, but it seems ng-reflect has a character cap of 30, so I'm not able to see the full expression. angular/angular#28786